### PR TITLE
Add support to show count of git staged files

### PR DESCRIPTION
**Pull request by CarlosMChica**

### CODE REVIEW

**Comment by CarlosMChica** [comment]
Are you still interested in this PR? I got 2 more ready waiting for this one to be merged. 

I'm very interested in the project and in adding full support for git status, but it looks like you may not have enough time to review the PRs. Would you be open to making me a maintainer, so that I can add the new features myself?

**Comment by rdnetto** [comment]
Sorry, my personal life has been quite hectic lately - I should have reviewed this weeks ago, and overall I definitely haven't given the time I should have.

Thanks for volunteering, I'm more than happy to give maintainer rights. You should be able to merge it now. :)

**Review by rdnetto — APPROVED** [review]

**Pull request merged**
